system-addon/jar.mn

```diff
@@ -17,6 +17,13 @@
   content/vendor/redux.js (./vendor/redux.js)
   content/vendor/react-redux.js (./vendor/react-redux.js)
   content/data/ (./data/*)
+#ifdef XP_MACOSX
+  content/data/content/activity-stream.css (./data/content/activity-stream-mac.css)
```

Comment:
The `content/data/ (./data/*)` line is going to package all of them, so most likely leading to unreferenced file test failure. Do we make a new "top level" directory outside of data? `css/` ?

Reply:
I suppose you can create a new directory in this PR, and I'll update it in https://github.com/mozilla/activity-stream/pull/3755 if we don't want to put css in `prerendered/static/activity-stream-mac.css`.

Alternatively, the only thing inside `data/` will be `data/content/` after I remove `locales.json` from the other PR #3755. So we could have a `data/css/*.css` that we jar.mn package appropriately… and now that only activity-stream.bundle.js is the only single file:
```
  content/data/content/activity-stream.bundle.js (./data/content/activity-stream.bundle.js)
  content/data/content/assets/ (./data/content/assets/*)
  content/data/content/tippytop/ (./data/content/tippytop/*)
#ifdef …
  content/data/content/activity-stream.css (./data/css/activity-stream-mac.css)
```

And I suppose with that change, I could move prerendered to `data/prerendered/`.

(I suppose we could move things "up" a directory, i.e., ./data/{activity-stream.bundle.js,assets,css,tippytop,prerendered} if we're going to CLOBBER anyway although history might be broken. Separately, we could expose the `resource://activity-stream/data/content/*` as just `resource://activity-stream/content/*`…)

Reply:
On a related note, should I be updating https://github.com/mozilla/activity-stream/pull/3775 (wow #3775 = package all locales vs #3755 = remove locales.json) and the related mozreview patch to use `resource://activity-stream/data/content/prerendered/locales/en-US` instead of `resource://activity-stream/prerendered/locales/en-US` ? if we move things to `./data/prerendered` ?

Reply:
I like the idea of moving stuff to top-level directories (content/data/content was kind of weird anyway)